inversion-api/src/main/java/io/inversion/utils/RestClient.java

```diff
@@ -860,6 +902,15 @@ public RestClient withName(String name) {
         return this;
     }
 
+    public String getCompression() {
```

Comment:
this is a bit confusing. Can we make this an enum?

Reply:
I copied a technique I saw in the Tomcat code.  There is a comment on the member prop saying that values can be "on", "off", "force" or the number of bytes over which you should compress. Probably right that it is confusing.  I split it out into two different props "compression" and "compressionMinSize"